.travis/test_06_script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
```

Comment:
14081 shouldn't just add this file

Reply:
yeah, it was created in 13863 originally but we missed it in #4223...